src/frontend/src/app/myprofile/myprofile.component.html

```diff
@@ -0,0 +1,49 @@
+<button class="leave-button" (click)="leave()">Back</button>
+<div align="center">
+  <h1 class="title">CodeChamp</h1>
+</div>
+<div align="center">
+  <h1 class="title">Player1</h1>
+</div>
+<section class="container">
+  <p class="stats-title">Stats</p>
+  <p class="stats-title">Match History</p>
+  <div class="left-half">
+    <mat-card class="stats-card">
+      <h1 class="stat-text1">Wins: 10</h1>
```

Comment:
Not needed for now but would make it easier for the feature - add those variables to a type and reference them here (hardcode the actual values in the .ts for now, or hardcode them in a service which will be replaced with a network call later).

Reply:
![image](https://user-images.githubusercontent.com/25267171/213953243-fdc36cdf-d583-4daf-8cb9-736d7ea2af1c.png)
![image](https://user-images.githubusercontent.com/25267171/213953258-0afdf426-c68b-4f85-a6a3-fec0bfd13dc2.png)